### PR TITLE
maint: add setuptools_scm support and auto-pypi uploads

### PR DESCRIPTION
**Pull request by steven-murray**
Adds setuptools_scm support, and tries uploading to pypi on git tag.

### CODE REVIEW

File: aipy/_version.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# file generated by setuptools_scm
+# don't change, don't track in version control
```
Comment (mkolopanis) [inline aipy/_version.py]: well. let's untrack this then.

File: setup.py [hunk 2]
```diff
@@ -6,36 +6,6 @@
 
 import os, glob, numpy, subprocess, sys
 
-PY2 = sys.version_info.major < 3
```
Comment (mkolopanis) [inline setup.py:9]: this change basically requires Python 3? I'm personally not opposed but I think this repo does need ongoing python 2 support. @plaplant can you comment?

**Review by mkolopanis — COMMENTED** [review]
I think there's some good stuff here, But i would recommend we move the auto-labeler to a different PR to disentangle the ideas. Also did we make sure all the info from PKG-INFO is in the setup.py or elsewhere? it's tough to tell just from the diffs

File: setup.py (hunk 2 above)
Comment (steven-murray) [reply setup.py:9]: I added Py2 support back in. Can't be bothered with that here.

File: aipy/_version.py (hunk 1 above)
Comment (steven-murray) [reply aipy/_version.py]: Woops, yup done.

**Comment by steven-murray** [comment]
@mkolopanis addressed all that and opened #63. PKG-INFO info was old and mostly wrong.

**Review by mkolopanis — APPROVED** [review]

**Pull request merged**